tests/framework/utils/k8s_helper.go

```diff
@@ -644,3 +646,43 @@ func (k8sh *K8sHelper) IsRookInstalled() bool {
 
 	return false
 }
+
+//GetRookLogs captures logs from specified rook pod and writes it to specified file
+func (k8sh *K8sHelper) GetRookLogs(podAppName string, namespace string, filePath string, testName string) {
+	lines := int64(2000)
+	logOpts := &v1.PodLogOptions{TailLines: &lines}
+	listOpts := metav1.ListOptions{LabelSelector: "app=" + podAppName}
+
+	podList, err := k8sh.Clientset.Pods(namespace).List(listOpts)
+	if err != nil {
+		logger.Errorf("Cannot get logs for app : %v in namespace %v", podAppName, namespace)
```

Comment:
this should log out the error object too.  also, the message should be more specific about "cannot list pods..."